tendermint/src/abci/event.rs

```diff
@@ -18,7 +18,7 @@ pub struct Event {
     ///
     /// Tendermint calls this the `type`, but we use `kind` to avoid confusion
     /// with Rust types and follow Rust conventions.
-    #[serde(rename = "type")]
+    #[serde(rename = "type", default)]
```

Comment:
I'm not sure this is an optimal solution, because it's recreating the data format by hand rather than deriving it from the source of truth (the proto messages). See https://github.com/informalsystems/tendermint-rs/issues/1415#issuecomment-2081010722

Reply:
Went for the shortest fix to an immediate issue. But anyone else, please feel free to fix it differently.

Reply:
Let's go with this fix for now since we are already re-creating the data format by hand everywhere in tendermint-rs.

Maybe we can address this whole thing at once and move to ProtoJSON in a follow-up PR or in cometbft-rs.